mirage/seed_image/psf_generator.py

```diff
@@ -0,0 +1,160 @@
+#! /suyr/bin/env python
+
+"""
+Class to generate PSFs to be used with Mirage
```

Comment:
A small thing - would this file be better located in mirage/psf rather than mirage/seed_image?

Reply:
Before this branch is merged, I'm pretty sure that file should be deleted. It relies on the older-style PSF filenames (rather than files created by Shannon's functions), and loads them into a FittableImageModel instance.